### PR TITLE
Update prow-tests image generation instructions

### PR DESCRIPTION
**Pull request by albertomilan**
**What this PR does, why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes to reviewers**:

**User-visible changes in this PR**:

### CODE REVIEW

File: images/prow-tests/README.md [hunk 1]
```diff
@@ -28,19 +28,13 @@ something complicated, do some rudimentary exploration in the image by running
...
-is actually wrong.
-
-Once you are happy the :beta image is working, go to
-https://gcr.io/knative-tests/test-infra/prow-tests and add the label `stable` to
-your image with the `beta` tag. Monitor #test on Slack to see if any new
-complaints appear.
+knative/test-infra and get approval. Once merged, a [postsubmit Prow job](https://prow.knative.dev/?job=post-knative-test-infra-prow-tests-image-push) will
+get triggered and build the new image into [Google Container Registry](https://pantheon.corp.google.com/gcr/images/knative-tests/GLOBAL/test-infra/prow-tests?gcrImageListsize=30). After the job succeeds, confirm the image is available
+with the `beta` and `latest` labels in the registry.
+At midnight PST, another set of Prow jobs will be triggered to run the
```
Comment (coryrc) [inline images/prow-tests/README.md:34]: "Around" midnight PST

**Comment by coryrc** [comment]
other than one small caveat, looks great!

File: images/prow-tests/README.md (hunk 1 above)
Comment (albertomilan) [reply images/prow-tests/README.md:34]: I've checked that and that seems more accurate. Most, but not all of them,  are executed ~10:20PM, so results should be ready around midnight. Ty

**Pull request merged**